javascriptv3/example_code/README.rst

```diff
@@ -0,0 +1,43 @@
+###############################
+AWS SDK for JavaScript version 3 (v3) examples
+###############################
+
+These are examples for the AWS SDK for JavaScript (v3) public documentation.
```

Comment:
Not sure why "(v3)" is parenthetical here--the version number here seems fairly important.

Reply:
Removed parentheses around v3 after first instance on all README files